modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_1/UpgradeSchema.java

```diff
@@ -24,6 +26,23 @@
 
 	protected void alterTables() throws Exception {
 		try (LoggingTimer loggingTimer = new LoggingTimer()) {
+			if (!hasColumnType(
+					DDMFormInstanceTable.TABLE_NAME, "description",
+					"TEXT null")) {
+
+				alter(
+					DDMFormInstanceTable.class,
+					new AlterColumnType("description", "TEXT null"));
+			}
+
+			if (!hasColumnType(
+					DDMFormInstanceVersionTable.TABLE_NAME, "description",
+					"TEXT null")) {
+
+				alter(
+					DDMFormInstanceVersionTable.class,
+					new AlterColumnType("description", "TEXT null"));
+			}
```

Comment:
@rodrigopaulino, shouldn't this code change the column type to `CLOB`? I believe I didn't get. 😞

Reply:
Basically, Liferay translates "TEXT" to large character objects and it depends on the database you're using. When using, MySQL for example, it will be `LONGTEXT`. Using Oracle Database, it will be `CLOB`.